suite-common/redux-utils/src/createThunk.ts

```diff
@@ -1,9 +1,13 @@
-import { AsyncThunkOptions, AsyncThunkPayloadCreator, createAsyncThunk } from '@reduxjs/toolkit';
+import {
+    AsyncThunkOptions,
+    AsyncThunkPayloadCreator,
+    createAsyncThunk as createAsyncThunkReduxToolkit,
```

Comment:
Why rename it if there is no overshadow of variable names?

Reply:
Because it will throw error otherwise, because of new eslint rule 😅